src/scripts/items/EvolutionStone.ts

```diff
@@ -32,3 +32,17 @@ ItemList['Metal_coat']    = new EvolutionStone(GameConstants.StoneType.Metal_coa
 ItemList['Kings_rock']    = new EvolutionStone(GameConstants.StoneType.Kings_rock, 2500);
 ItemList['Upgrade']       = new EvolutionStone(GameConstants.StoneType.Upgrade, 2500);
 ItemList['Time_stone']    = new EvolutionStone(GameConstants.StoneType.Time_stone, 2500);
+ItemList['Soothe_bell']   = new EvolutionStone(GameConstants.StoneType.Soothe_bell, 2500);
+ItemList['Deepsea_tooth'] = new EvolutionStone(GameConstants.StoneType.Deepsea_tooth, 2500);
+ItemList['Deepsea_scale'] = new EvolutionStone(GameConstants.StoneType.Deepsea_scale, 2500);
+ItemList['Time_stone']    = new EvolutionStone(GameConstants.StoneType.Time_stone, 2500);
+ItemList['Dawn_stone']    = new EvolutionStone(GameConstants.StoneType.Dawn_stone, 2500);
+ItemList['Dusk_stone']    = new EvolutionStone(GameConstants.StoneType.Dusk_stone, 2500);
+ItemList['Shiny_stone']   = new EvolutionStone(GameConstants.StoneType.Shiny_stone, 2500);
+ItemList['Dubious_disc']  = new EvolutionStone(GameConstants.StoneType.Dubious_disc, 2500);
+ItemList['Electrizer']    = new EvolutionStone(GameConstants.StoneType.Electrizer, 2500);
+ItemList['Magmarizer']    = new EvolutionStone(GameConstants.StoneType.Magmarizer, 2500);
+ItemList['Protector']     = new EvolutionStone(GameConstants.StoneType.Protector, 2500);
+ItemList['Reaper_cloth']  = new EvolutionStone(GameConstants.StoneType.Reaper_cloth, 2500);
+ItemList['Razor_claw']    = new EvolutionStone(GameConstants.StoneType.Razor_claw, 2500);
+ItemList['Razor_fang']    = new EvolutionStone(GameConstants.StoneType.Razor_fang, 2500);
```

Comment:
Can you add a blank line at the end of the file,
It also looks like the Prism_scale is missing.